i18n/qml-translation.zh_CN.ts

```diff
@@ -183,20 +183,20 @@
     <message>
         <location filename="../qml/ttrss/sailfish/pages/ApiSettings.qml" line="109"/>
         <source>Additional HTTP Auth?</source>
-        <translation type="unfinished"/>
-    </message>
-    <message>
+        <translation/>附加 HTTP Auth？</translation>
```

Comment:
The xml syntax went a little wrong here ;) but nevertheless thank you for the translation update, i will release another update version in the coming days